src/routers/DelayRouter.js

```diff
@@ -14,13 +14,13 @@ class DelayRouter extends ApplicationRouter<Object> {
 
     async content(req): Promise<any> {
         const { stopID, tripID } = req.query;
-        LogUtils.log({ category: 'DelayRouter', stopID, tripID });
-        const res = RealtimeFeedUtils.getTrackingInformation(
+        LogUtils.log({
+            category: 'DelayRouterRTF',
```

Comment:
probably should just change to 'DelayRouter' since we're not returning the rtf
```suggestion
            category: 'DelayRouter',
```